fastq_filtrator.py

```diff
@@ -0,0 +1,200 @@
+# Here is fastq filtrator
+
+
+# GC-filtering
+def filter_gc(input_list, gc_bounds, save_filtered):
+    input_list_1 = list()
+    output_list_failed_1 = list()
+    for n in range(0, len(input_list)):
+        if n % 4 == 1:
+            gc = 100*2*input_list[n].count('G')/len(input_list[n])
+            if gc > int(gc_bounds[0]) and gc < int(gc_bounds[1]):

```

Comment:
1. Все интервалы в ДЗ должны включать границы.
2. Можно немного упростить
```suggestion
            if int(gc_bounds[0]) <= gc <= int(gc_bounds[1]):
```